lms/djangoapps/grades/new/course_grade.py

```diff
@@ -112,6 +138,9 @@ def summary(self):
         # doesn't get displayed differently than it gets grades
         grade_summary['percent'] = self.percent
         grade_summary['grade'] = self.letter_grade
+
+        # These two fields are stored separately from percent and grade. The should match unless accessed in between
+        # course and subsection grade updates. If that happens we re-populate them here.
```

Comment:
This comment has a typo (`The should match`), and confuses me.  

1. How are they stored separately?  That's either trivially obvious, in that they are different attributes, or confusingly false, in that they are stored on the same grade_summary object.  
2. What should they match? Each other?  Then we should have just one entry.  The percent and letter grade values?  Then be explicit about that, and what the match looks like.
3. We repopulate them whether or not anything happens, not based on a condition.

After writing all that, I think I understand this to mean that there is a race condition that could cause these to get out of sync, (with each other?) and recalculating them here resolves that issue. Is that accurate? Can you rewrite this comment for clarity?